app/src/main/java/org/mozilla/fenix/utils/ManufacturerCodes.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.utils
+
+import android.os.Build
+
+/**
+ * A listing of codes returned by [android.os.Build.MANUFACTURER] for different manufacturers.
+ * While we try to get the casing accurate, it may be good to use .equals(str, ignoreCase = true)
+ * to do the comparison.
+ */
+object ManufacturerCodes {
+    const val HUAWEI: String = "HUAWEI"
```

Comment:
just a small nit: We use `String` for 2 of the variables and 2 don't have it.

Reply:
You're right that this was sloppy but I don't want to take the time to fix it up, wait for CI, etc. so I'll leave it in. 😅 I can clean it up next time I'm in this area.